src/main/java/uk/gov/justice/hmpps/datacompliance/client/duplicate/detection/DuplicateDetectionQueryFactory.java

```diff
@@ -35,7 +35,7 @@
     public DuplicateDetectionQueryFactory(@Value("${duplicate.detection.athena.database}") final String database,
                                           @Value("${duplicate.detection.athena.table}") final String table,
                                           @Value("${duplicate.detection.athena.output.location}") final String outputLocation,
-                                          @Value("${duplicate.detection.match.score.threshold:0.2}") final double matchScoreThreshold) {
+                                          @Value("${duplicate.detection.match.score.threshold:0.75}") final double matchScoreThreshold) {
```

Comment:
Increasing to a much more reasonable default threshold based on preprod analysis